store/src/java/com/zimbra/cs/account/ldap/LdapProvisioning.java

```diff
@@ -9451,6 +9483,45 @@ public long countObjects(CountObjectsType type, Domain domain, UCService ucServi
         return num;
     }
 
+    @Override
+    public long countObjects(CountObjectsType type, Object... args) throws ServiceException {
+        ZLdapFilter filter = null;
```

Comment:
Please add null checks to ensure that the arguments passed are not null and are of the expected types and quantities.

Reply:
Done